desc/configuration.py

```diff
@@ -312,19 +325,58 @@ def __init__(
         else:
             raise TypeError("Got unknown pressure profile {}".format(pressure))
 
+        # default profile
+        if iota is None and current is None:
+            warnings.warn(
```

Comment:
Not sure if we need an explicit warning if the default option is clear from the docstring

Reply:
Next commit removes the warning and removes the incorrect statement that iota defaults to 0 from the docstring.